### PR TITLE
Milestone 5

### PR DESCRIPTION
**Pull request by kazumaki**
Connect calculate and operate modules to app component to show the results.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+
+const mainContainerStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: 700,
+};
+
```
Comment (shloch) [inline src/components/App.js]: ### [Optional]
You can center calculator to be in middle of screen
```
    margin: auto;
```

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const bigTwo = new Big(numberTwo);
+
+  if (operation === '+') {
+    result = bigOne.plus(bigTwo);
+  } else if (operation === '-') {
+    result = bigOne.minus(bigTwo);
+  } else if (operation === 'X') {
+    result = bigOne.times(bigTwo);
+  } else if (operation === '÷') {
+    result = bigOne.div(bigTwo);
```
Comment (shloch) [inline src/logic/operate.js]: ### [Required]
You've not managed division by ZERO...please handle this
calc

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @kazumaki , 

Great job...everything works fine
You however have one small bug to rectify (mentioned below) and we'll be good

### Happy coding

File: src/logic/operate.js (hunk 2 above)
Comment (shloch) [reply src/logic/operate.js]: As you can see from gif, in beginning, normal division wasn;t working, i dont know why....then i refreshed, works fine...except for DIVISION BY  ZERO

Also please try to increase the font size of buttons a little.....looks smaller and you got space, thanks

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @kazumaki good job starting this and also working with the previous reviewer to fix the bug he pointed out :+1: but it seems there are more breaking changes now :slightly_smiling_face:  I will add a video link [here](https://www.loom.com/share/76d7e6763da04b0489b01addc756c530) please look at it 

Also, the division by zero error is not corrected either this shows in the video too 
Using the previous answer to make a further calculation is not well supported, for example, type 8 * 3 would give 24, if the user then types + 6 it should give 30 but the answer remains 6. 
This and some other issues are shown in the video linked here 

Please make changes and request for another review :slightly_smiling_face: 

**RAPHAEL NORIODE** :mask:

**Review by bellom — APPROVED** [review]
## Hi @kazumaki  :clap: 

**Great work making all the necessary corrections from the previous review.**

NOTE: It would be better to return a message on the screen whenever users divide with zero just like a normal calculator, in your case you return `0`

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**